userspace/libscap/scap_procs.c

```diff
@@ -561,6 +564,55 @@ int32_t scap_proc_fill_loginuid(scap_t *handle, struct scap_threadinfo* tinfo, c
 	}
 }
 
+int32_t scap_proc_fill_exe_writable(scap_t* handle, struct scap_threadinfo* tinfo,  uint32_t uid, uint32_t gid, const char *procdirname, const char *exetarget)
```

Comment:
Don't you need to save this information to the process list as well? I would think with these changes, a process would have the exe writable flag tracked if read from proc but it would get lost when saved to a capture file.

Reply:
Good point. I didn't try writing a scap file and reloading it, will check.

Reply:
It should work now.